docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+COPY restic /usr/bin
```

Comment:
> This will failed because restic is in the root directory of repo while Dockerfile is put in docker.

```
$ docker run restic/restic
restic is a backup program which allows saving multiple revisions of files and
directories in an encrypted repository stored on different backends.

Usage:
  restic [command]
```

why it works for me?